<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LTPCHE102076955" name="com.pom.crimson.testcases.momentlog.high.EditMomentPageTest" tests="6" failures="1" timestamp="2021-01-03T08:46:26 IST" time="968.435" errors="0">
  <testcase name="deleteAllData_EditMomentPageTest" time="38.405" classname="com.pom.crimson.testcases.momentlog.high.EditMomentPageTest"/>
  <system-out/>
  <testcase name="verifyCrossBtn_EditMomentPageTest" time="93.308" classname="com.pom.crimson.testcases.momentlog.high.EditMomentPageTest"/>
  <system-out/>
  <testcase name="verifyRemoveMilestone_EditMomentPageTest" time="229.040" classname="com.pom.crimson.testcases.momentlog.high.EditMomentPageTest"/>
  <system-out/>
  <testcase name="verifyRemoveNote_EditMomentPageTest" time="230.100" classname="com.pom.crimson.testcases.momentlog.high.EditMomentPageTest"/>
  <system-out/>
  <testcase name="verifyRemoveHeightWeight_EditMomentPageTest" time="271.459" classname="com.pom.crimson.testcases.momentlog.high.EditMomentPageTest"/>
  <system-out/>
  <testcase name="verifyEditDate_EditMomentPageTest" time="106.123" classname="com.pom.crimson.testcases.momentlog.high.EditMomentPageTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Check if date is saved in backend expected [2021-01-03T00:00:00Z] but found [2021-01-01T00:00:00Z]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Check if date is saved in backend expected [2021-01-03T00:00:00Z] but found [2021-01-01T00:00:00Z]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.pom.crimson.testcases.momentlog.high.EditMomentPageTest.verifyEditDate_EditMomentPageTest(EditMomentPageTest.java:380)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyEditDate_EditMomentPageTest -->
  <system-out/>
</testsuite> <!-- com.pom.crimson.testcases.momentlog.high.EditMomentPageTest -->
